CRM/Admin/Form/Job.php

```diff
@@ -194,6 +204,37 @@ public function postProcess() {
     $dao->description = $values['description'];
     $dao->is_active = CRM_Utils_Array::value('is_active', $values, 0);
 
+    // CRM-17686
+    $dt = '';
+    if (!empty($values['scheduled_run_date'])) {
+      $dt = $values['scheduled_run_date'] . ' ';;
+    }
+    if (!empty($values['scheduled_run_date_time'])) {
+      $dt .= $values['scheduled_run_date_time'];
+    }
+    $ts = strtotime(trim($dt));
```

Comment:
And this can be simplified to

    $ts = strtotime($values['scheduled_run_date']);